jest.config.js

```diff
@@ -0,0 +1,11 @@
+/** @typedef {import('ts-jest')} */
+/** @type {import('@jest/types').Config.InitialOptions} */
+module.exports = {
+  preset: 'ts-jest',
```

Comment:
once we have unit tests for `js` files, the preset value should be `ts-jest/presets/js-with-ts` and also add `allowJs: true` to `tsconfig.spec.json`

Reply:
Or maybe we should make all files as `ts`, including serializers. My idea is: group all serializers in one folder, another folder for transformers and these folders will have an `index.ts`.

In the root index, we can link to those `index` to export them to public.

If we do so, we should do in the next major release.

Reply:
I think we should do one minor release which includes zone.js fix + preset config change. Then after that we can proceed with the next major